packages/devtools/frigg-cli/environmentVariables.js

```diff
@@ -0,0 +1,134 @@
+const fs = require('fs');
+const dotenv = require('dotenv');
+const { readFileSync, writeFileSync, existsSync } = require('fs');
+const { logInfo } = require('./logger');
+const { resolve } = require('node:path');
+const inquirer = require('inquirer');
+
+const { parse } = require('@babel/parser');
+const traverse = require('@babel/traverse').default;
+
+const extractRawEnvVariables = (modulePath) => {
+    const filePath = resolve(modulePath, 'definition.js');
+
+    const fileContent = fs.readFileSync(filePath, 'utf-8');
+    const ast = parse(fileContent, {
+        sourceType: 'module',
+        plugins: ['jsx', 'typescript'], // Add more plugins if needed
+    });
+
+    const envVariables = {};
+
+    traverse(ast, {
+        ObjectProperty(path) {
+            if (path.node.key.name === 'env') {
+                path.node.value.properties.forEach((prop) => {
+                    const key = prop.key.name;
+                    if (prop.value.type === 'MemberExpression') {
+                        const property = prop.value.property.name;
+                        envVariables[key] = `${property}`;
+                    } else if (prop.value.type === 'TemplateLiteral') {
+                        // Handle template literals
+                        const expressions = prop.value.expressions.map((exp) =>
+                            exp.type === 'MemberExpression'
+                                ? `${exp.property.name}`
+                                : exp.name
+                        );
+                        envVariables[key] = expressions.join('');
+                    }
+                });
+            }
+        },
+    });
```

Comment:
Did you copy that from somewhere ? 😂
Hard to follow

Reply:
It was a generated code snippet. Can refactor later for comprehension 👍